libafl_frida/src/cmplog_rt.rs

```diff
@@ -0,0 +1,187 @@
+use dynasmrt::{dynasm, DynasmApi, DynasmLabelApi};
+use nix::{
+    libc::{memmove, memset},
+    sys::mman::{mmap, MapFlags, ProtFlags},
+};
+use std::ffi::c_void;
+
+// TODO compile time flag
+//pub const CMPLOG_MAP_W: usize = 65536;
+pub const CMPLOG_MAP_W: usize = 8000;
```

Comment:
This code is duplicated from libafl_targets. Btw, I reworked CmpLog in branch (cmplog branch), there you can find directly some function that you can call, don't rely on sancov functions for frida

Reply:
did you meant __libafl_targets_cmplog?

Reply:
You can use directly libafl_targets::cmplog (https://github.com/AFLplusplus/LibAFL/blob/main/libafl_targets/src/cmplog.rs). For the instrumentation routine, yes it is __libafl_target_cmplog and it is in C (https://github.com/AFLplusplus/LibAFL/blob/main/libafl_targets/src/cmplog.h). To expose it to rust, you can simply create a non static stub that call this static function and use it from Rust as extern "C"